sdk/monitor/monitor-opentelemetry/src/metrics/utils.ts

```diff
@@ -197,6 +197,19 @@ export function getProcessorTimeNormalized(
   return (usageDifMs / elapsedTimeMs / numCpus) * 100;
 }
 
+// This function can get the cpu %, but it assumes that after this function is called,
+// that the process.hrtime.bigint() & process.cpuUsage() are called/stored to be used as the
+// parameters for the next call.
+export function getProcessorTime(lastHrTime: bigint, lastCpuUsage: NodeJS.CpuUsage): number {
+  const usageDif = process.cpuUsage(lastCpuUsage);
```

Comment:
Do we need to validate that `process` and `process.cpuUsage` exist -- otherwise they will throw

Reply:
There are scenarios where the `process` object could be temporarily undefined, so this is a good case to account for. The try-catch surrounding `onEnd()` in the `AzureMonitorSpanProcessor` would stop this from fully crashing the app, but we can provide some more specifics on the issue.